pkg/cache/cache.go

```diff
@@ -395,6 +399,7 @@ func (c *Cache) AddClusterQueue(ctx context.Context, cq *kueue.ClusterQueue) err
 			key:                qKey,
 			reservingWorkloads: 0,
 			admittedWorkloads:  0,
+			labels:             q.Labels,
```

Comment:
Should this be behind the LocalQueueMetrics flag - or are we confident low risk of side effects (memory usage blowup if a LQ had many labels) that may need to be disabled? I suppose the latter (no need for flag)

Reply:
I'll add a check to see if LQ metrics is enabled